cmd/secrets/read_secret.go

```diff
@@ -0,0 +1,145 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2019 Datadog, Inc.
+
+// +build secrets
+
+package secrets
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	maxSecretFileSize    = 1024
+	compatibleMajVersion = "1"
+)
+
+type secret struct {
```

Comment:
Same here, why not use the struct in secret ?

Reply:
Can do that if you don't mind my prior edits for json annotations of that struct.

Reply:
I think the edits on the JSON annotations where fine (I didn't saw any breaking change).